src/lwc/geFormField/__tests__/geFormField.test.js

```diff
@@ -54,16 +89,181 @@ describe('c-ge-form-field', () => {
+
+        document.body.appendChild(element);
+
+        return Promise.resolve().then(() => {
+            expect(element.isPicklist).toBeTruthy();
+            expect(getSubComponentCombobox(element).options.length).toEqual(2);
+            expect(getSubComponentCombobox(element).options).toContainOptions([
+                '000000000000000001',
+                '000000000000000002'
+            ]);
+        });
+    });
+
+    it('has correct default value for standard picklists', async () => {
+        const element = createStandardPicklistElement();
+
+        getObjectInfoAdapter.emit(mockSObjectDescribeInfo);
+        getPicklistValuesWireAdapter.emit(mockGetPicklistValues);
+
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        return Promise.resolve().then(() => {
+            expect(picklist.value).toBe('--None--');
+        });
     });
 
+    it('has correct default value for record type id picklists', async () => {
+        const element = createRecordTypeIdPicklistElement();
+
+        getObjectInfoAdapter.emit(mockSObjectDescribeInfo);
+
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        return Promise.resolve().then(() => {
+            expect(picklist.value).toBe('');
+        });
+    });
+
+    it('has correct value when changed in form state', async () => {
+        const element = createStandardPicklistElement();
+
+        getObjectInfoAdapter.emit(mockSObjectDescribeInfo);
+        getPicklistValuesWireAdapter.emit(mockGetPicklistValues);
+
+        document.body.appendChild(element);
+
+        const picklist = getSubComponentCombobox(element);
+
+        const picklistChangeHandler = jest.fn();
+        element.addEventListener('formfieldchange', picklistChangeHandler);
+
+        return Promise.resolve()
+            .then(() => {
+                expect(picklist.value).toBe('--None--');
+
+                dispatchChangeEvent(picklist, 'Picklist_Option_1');
+
+                jest.runOnlyPendingTimers();
+
+                expect(picklistChangeHandler).toBeCalled();
+                expect(picklistChangeHandler.mock.calls[0][0].detail.value).toStrictEqual('Picklist_Option_1');
+
+                mockFormState(element, 'Picklist_Option_1');
+            })
+            .then(() => {
+                expect(picklist.value).toBe('Picklist_Option_1');
+
+                dispatchChangeEvent(picklist, 'Picklist_Option_2');
+
+                jest.runOnlyPendingTimers();
+
+                expect(picklistChangeHandler).toBeCalled();
+                expect(picklistChangeHandler.mock.calls[1][0].detail.value).toStrictEqual('Picklist_Option_2');
+
+                mockFormState(element, 'Picklist_Option_2');
+            })
+            .then(() => {
+                expect(picklist.value).toBe('Picklist_Option_2');
+            });
+    });
+
+});
+
+expect.extend({
```

Comment:
:+1: cool use of expect.extend.

Reply:
The lightning-global repo has a bunch of these extensions and other useful utils. Only pulled the ones that seemed immediately useful for these tests 🙏